app/code/local/Magmodules/Webwinkelconnect/Model/Api.php

```diff
@@ -159,4 +155,33 @@ public function getStoreIds()
         return $storeIds;
     }
 
+    private function getOrderProducts(Mage_Sales_Model_Order $order): array {
+        $products = [];
+        foreach ($order->getAllItems() as $item) {
+            $product = $item->getProduct();
+            $products[] = [
+                'name' => $product->getName(),
+                'url' => $product->getProductUrl(),
+                'id' => $product->getEntityId(),
```

Comment:
Wait, is this really the correct ID? Since in another place the entity ID is the ID of the "type" of entity, not the primary key of the item itself. Is that different here?